docs/sphinx/pom.xml

```diff
@@ -59,7 +59,6 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>exec-maven-plugin</artifactId>
-        <version>1.2.1</version>
```

Comment:
That is the impression I get from `mvn help:effective-pom`.

Note that `components/formats-{api,bsd}/pom.xml` also make use of a `[1.2.1,)` version specifier.